pkg/apis/core/validation/validation.go

```diff
@@ -3557,13 +3557,10 @@ func validatePodAffinityTermNamespaceSelector(podAffinityTerm core.PodAffinityTe
 		return nil
 	}
 
-	allErrs := field.ErrorList{}
-	for i, m := range nsSelector.MatchExpressions {
-		if m.Operator != metav1.LabelSelectorOpIn {
-			allErrs = append(allErrs, field.NotSupported(fldPath.Child("matchExpressions").Index(i).Child("operator"), m.Operator, []string{"In"}))
-		}
+	allErrs := unversionedvalidation.ValidateLabelSelector(nsSelector, fldPath)
+	if metav1.LabelSelectorIsEmpty(nsSelector) {
+		allErrs = append(allErrs, field.Invalid(fldPath, "{}", "can not be empty"))
```

Comment:
nit: just pass the object to the function, as we do for any other errors

Reply:
done.